### PR TITLE
release-notes: add release notes for the v92 release

### PR DESCRIPTION
**Pull request by ricardosalveti**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Release notes for the v92 release.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by ricardosalveti** [comment]
The actual link to the v92 release notes is broken as that will only be available after the release is actually published, we need to coordinate this one better (we might remove for now and later release just the docs with the updated link).

**Comment by doanac** [comment]
Docs for 2b35f0501def3a10a5fac316231b15eea14aa806 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2532/docs/artifacts/html/index.html

**Comment by ricardosalveti** [comment]
@kprosise we might not be able to enforce LmP when that is part of a recipe name, such as linux-lmp or similar.

**Comment by doanac** [comment]
Docs for 3f0a7e033e764f6fe3d38ee445e039861ce83c23 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2533/docs/artifacts/html/index.html

**Comment by ricardosalveti** [comment]
Some of the dash errors are due the original tag name used (e.g. lf-6.1.36-2.1.0).

**Comment by doanac** [comment]
Docs for be8d14bcca1a20b6aa06964bd42238a97c0633d0 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2534/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
@ricardosalveti don't worry about any linter checks, they are general guidelines.
I will give the document a full review in the morning!

**Comment by doanac** [comment]
Docs for 8b3de772d06b1caf1e41c704fc6fc95c8c531b39 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2535/docs/artifacts/html/index.html

**Review by quaresmajose — APPROVED** [review]

File: source/release-notes/release-notes.rst [hunk 1]
```diff
@@ -1,27 +1,77 @@
...
-You can view the full changelog LINK TO CHANGELOG for greater detail
+.. note::
+    You can view the full `v92 changelog`_ for greater detail.
 
-Important Changes From vXX
+Important Changes From V91
 ---------------------------
 
-Quality of life changes which may effect how a customer interacts with a Factory
+Things to be aware when updating LmP from the v91 release:
```
Comment (kprosise) [inline source/release-notes/release-notes.rst:12]:
```suggestion
Things to be aware of when updating LmP from the v91 release:
```

File: source/release-notes/release-notes.rst [hunk 2]
```diff
@@ -1,27 +1,77 @@
...
+ * NXP BSP updated to the lf-6.1.36-2.1.0 release:
+
+   * U-boot had a major update from 2022.04 to 2023.04, porting work and configuration changes might be required
+   * OP-TEE updated from 3.20 to 3.21, requiring ELE firmware updates on both i.MX93 and i.MX8ULP targets
+   * Base BSP kernel updated from 6.1 to 6.1.36, with additional fixes and improved hardware support
+ * TI BSP updated to the 09.00.00.009 release, but without major changes when comparing with 09.00.00.005
+ * NVIDIA Tegra BSP updated to the L4T R35.4.1 release, still based on JetPack 5
+ * STM32MP15 flashlayout definition is now generated dynamically, which might require changes depending on the level of customization
+
+Please also make sure to check the respective vendor BSP release notes for more information.
```
Comment (kprosise) [inline source/release-notes/release-notes.rst:24]:
```suggestion
Please also check the respective vendor BSP release notes for more information.
```

**Review by kprosise — APPROVED** [review]
Just a couple small suggestions, otherwise it LGTM!

**Review by mike-sul — APPROVED** [review]

**Comment by doanac** [comment]
Docs for 68a5cda393e088d73109cf4eecff6d5f5740b616 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2539/docs/artifacts/html/index.html

**Comment by ricardosalveti** [comment]
@kprosise please merge if you are happy with it :-)

**Pull request merged**